frontend/js/src/explore/artist-similarity/SimilarArtistsGraph.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from "react";
+import {
+  ResponsiveNetwork,
+  NodeProps,
+  NodeTooltipProps,
+  NetworkSvgProps,
+} from "@nivo/network";
+import { animated, to } from "@react-spring/web";
+import type { NodeType, LinkType, GraphDataType } from "./Data";
+
+interface GraphProps {
+  data: GraphDataType;
+  onArtistChange: (artist_mbid: string) => void;
+  background: string;
+}
+
+type OmitHeightWidth<T> = Omit<T, "height" | "width">;
+
+const MAX_LINES = 2;
+const MAX_WORD_LENGTH = 10;
+
+function CustomNodeComponent({
+  node,
+  animated: animatedProps,
+  onClick,
+  onMouseEnter,
+  onMouseMove,
+  onMouseLeave,
+}: NodeProps<NodeType>) {
+  return (
+    <animated.g
```

Comment:
It would be nice to give these elements a class name so that we can target them.

For example, adding `cursor: pointer`  one these group elements would make it clearer that the graph is interactive